assets/src/dashboard/components/detailViewNavBar/index.js

```diff
@@ -79,11 +79,18 @@ const CapitalizedButton = styled(Button)`
   text-transform: uppercase;
 `;
 
-export function DetailViewNavBar({ handleCta, handleBookmarkClick, ctaText }) {
+export function DetailViewNavBar({
+  closeViewAriaLabel = __('Close', 'web-stories'),
+  handleCta,
+  handleBookmarkClick,
+  ctaText,
+}) {
   return (
     <Nav>
       <Container>
-        <CloseLink href={parentRoute()}>{__('Close', 'web-stories')}</CloseLink>
+        <CloseLink aria-label={closeViewAriaLabel} href={parentRoute()}>
+          {__('Close', 'web-stories')}
+        </CloseLink>
```

Comment:
What's the reason for not inlining the aria-label here?

Reply:
This nav component is in `/components` and the link could theoretically go to other views, so i am passing it in to keep the context tied to the view and not have it seep into components that should be reusable. Alternatively, this detailViewNavBar could really just be within the template detail view not /components now that we have more of the dashboard scaffolded out. 

Honestly, i'm just trying to keep this small and in scope and keep the components from becoming more opinionated.